modules/gdnative/SCsub

```diff
@@ -33,14 +33,39 @@ def _build_gdnative_api_struct_header(api):
         '',
         '#ifdef __cplusplus',
         'extern "C" {',
-        '#endif',
+        '#endif'
```

Comment:
The comma is still needed here, no?

Reply:
Oh yeah, for some reason it's not an error so I didn't notice. Will fix.